1.20.4 Support - Plugin not load

#### *Complete* output of the `ncp version` command (please prefer text here)

#### Short description of the issue and how to reproduce (is it random / always happens / side conditions ?)

#### Have you made any (noteworthy) changes to the default configuration of NCP?

#### Do you run any other special plugins that affect game mechanics next to NC+ (skills, machines, adding/changing blocks, other anti-cheating)?

#### Provide additional information: for bypasses, vulnerabilities and reproducible issues, we request a [debug log. ](https://github.com/Updated-NoCheatPlus/Docs/blob/master/Development/Debugging.md#on-the-fly-debug-output-for-individual-players) Use [GitHub gists](https://gist.github.com/) or [PasteBin](http://pastebin.com/) or similar for errors and huge logs, or attach a (zip-) file here.

```
[01:58:59 ERROR]: [ModernPluginLoadingStrategy] Could not load plugin 'NoCheatPlus.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: Exception initializing main class `fr.neatmonster.nocheatplus.NoCheatPlus'
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:106) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.provider.type.spigot.SpigotPluginProvider.createInstance(SpigotPluginProvider.java:123) ~[paper-1.20.4.jar:git-Paper-324]
        at io.papermc.paper.plugin.provider.type.spigot.SpigotPluginProvider.createInstance(SpigotPluginProvider.java:35) ~[paper-1.20.4.jar:git-Paper-324]
        at io.papermc.paper.plugin.entrypoint.strategy.modern.ModernPluginLoadingStrategy.loadProviders(ModernPluginLoadingStrategy.java:116) ~[paper-1.20.4.jar:git-Paper-324]
        at io.papermc.paper.plugin.storage.SimpleProviderStorage.enter(SimpleProviderStorage.java:38) ~[paper-1.20.4.jar:git-Paper-324]
        at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enter(LaunchEntryPointHandler.java:36) ~[paper-1.20.4.jar:git-Paper-324]
        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.loadPlugins(CraftServer.java:509) ~[paper-1.20.4.jar:git-Paper-324]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:273) ~[paper-1.20.4.jar:git-Paper-324]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1132) ~[paper-1.20.4.jar:git-Paper-324]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:321) ~[paper-1.20.4.jar:git-Paper-324]
        at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: java.lang.reflect.InvocationTargetException
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:98) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        ... 10 more
Caused by: java.lang.ExceptionInInitializerError
        at fr.neatmonster.nocheatplus.config.DefaultConfig.<init>(DefaultConfig.java:193) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.worlds.WorldDataManager.createDefaultWorldData(WorldDataManager.java:202) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.worlds.WorldDataManager.<init>(WorldDataManager.java:142) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.NoCheatPlus.<init>(NoCheatPlus.java:191) ~[NoCheatPlus.jar:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:98) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        ... 10 more
Caused by: java.lang.NullPointerException: Material not present: long_grass, grass
        at fr.neatmonster.nocheatplus.compat.BridgeMaterial.getFirstNotNull(BridgeMaterial.java:129) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.compat.BridgeMaterial.<clinit>(BridgeMaterial.java:236) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.config.DefaultConfig.<init>(DefaultConfig.java:193) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.worlds.WorldDataManager.createDefaultWorldData(WorldDataManager.java:202) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.worlds.WorldDataManager.<init>(WorldDataManager.java:142) ~[NoCheatPlus.jar:?]
        at fr.neatmonster.nocheatplus.NoCheatPlus.<init>(NoCheatPlus.java:191) ~[NoCheatPlus.jar:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:98) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        ... 10 more
```

--- a/NCPCore/src/main/java/fr/neatmonster/nocheatplus/compat/BridgeMaterial.java
+++ b/NCPCore/src/main/java/fr/neatmonster/nocheatplus/compat/BridgeMaterial.java
@@ -233,7 +233,7 @@ public class BridgeMaterial {
     public static final Material FIREWORK_ROCKET = getFirst("firework_rocket", "firework");
 
     /** Passable (long) grass block. */
-    public static final Material GRASS = getFirstNotNull("long_grass", "grass");
+    public static final Material GRASS = getFirstNotNull("short_grass", "long_grass", "grass");
 
     /** Classic dirt-like grass block. */
     public static final Material GRASS_BLOCK = getFirstNotNull("grass_block", "grass");

